### ISSUE TITLE
!pip install lxml problem!

### ISSUE THREAD

**Issue by gauravbagul003** [nicknochnack/TFODCourse#116]
installation of lxml is not happening  the error is popping saying to install Microsoft Visual C++ 14.0 And even after installing Microsoft Visual C++ 14.0. The error is not going.  Everything was going smoothly until this error showed up! :|

**Comment by sabbir2609** [comment]
can you show a screenshot of that error?

**Comment by gauravbagul003** [comment]
i don't have screenshot.

**Comment by sabbir2609** [comment]
try installing Microsoft Visual Studio Build Tools 2019

**Comment by gauravbagul003** [comment]
let me try again i will let you know if there is same problem
